web/packages/extension/src/content.ts

```diff
@@ -173,6 +173,9 @@ function isXMLDocument(): boolean {
             warnOnUnsupportedContent: options.warnOnUnsupportedContent,
             logLevel: options.logLevel,
             showSwfDownload: options.showSwfDownload,
+            autoplay: options.autostart ? "on" : "auto",
```

Comment:
What's the difference between "autoplay" and "autostart"? Is this just a typo perhaps?

Reply:
autostart controls unmuteOverlay, autoplay, and splashScreen options simultaneously.

Reply:
Right. Was just Ctrl+F-ing wrong, sorry.